### DISCUSSION TITLE
New version of AraBert_output_Embeddings_PyTorch.ipynb

### DISCUSSION THREAD

**Discussion by Leenarj** [aub-mind/arabert#71] [Q&A] [answered]
Hi,
Thank you for the hard work. 
I want to ask if can get the new version of [AraBert_output_Embeddings_PyTorch.ipynb](https://github.com/aub-mind/arabert/blob/master/examples/old/AraBert_output_Embeddings_PyTorch.ipynb).

regards

**Comment by WissamAntoun** [comment]
Sure, I'll work on it during the weekend

**Comment by WissamAntoun** [comment, marked as answer]
Here it is https://colab.research.google.com/drive/1Xk8IGcwGTr-YNlB6lhl8jYdIU_M7Mcjx

**Reply by Leenarj** [reply]
Thank you very much 🙏

**Reply by WissamAntoun** [reply]
is the shared link working now?

**Reply by Leenarj** [reply]
Yess thank you
